### PR TITLE
Fix chapters with negative start time

### PR DESCRIPTION
**Pull request by lukasf**
This solves the file with broken chapter entries from #185.

### CODE REVIEW

**Comment by mcosmin222** [comment]
Looks good!

**Comment by lukasf** [comment]
Start Duration is already negative, so I have to add it instead of subtracting it. Works well with the file.

**Comment by mcosmin222** [comment]
This might cause chapters to overlap though. I wonder if that even matters.

**Comment by mcosmin222** [comment]
Actually, shouldn't we handle the case in which duration is also negative? We could put a dummy duration like 1 or something, instead of skipping.

**Comment by lukasf** [comment]
Well I don't know. Chapters with negative duration sound pretty broken. I would not try to fixup too much here. Negative start time might be a more common case, when start logos / copyright notices were cut off from a movie.

File: FFmpegInterop/FFmpegInteropMSS.cpp [hunk 1]
```diff
@@ -966,7 +966,7 @@ HRESULT FFmpegInteropMSS::InitFFmpegContext()
 						// cut off negative start times
 						if (start.Duration < 0)
 						{
-							duration.Duration -= start.Duration;
+							duration.Duration += start.Duration;
 							start.Duration = 0;
```
Comment (mcosmin222) [inline FFmpegInterop/FFmpegInteropMSS.cpp:970]: ah, sorry, missed this :(

**Comment by mcosmin222** [comment]
Ok i think this is ready for the great merge.

**Pull request merged**